LiteCore/Database/DatabaseImpl+Upgrade.cc

```diff
@@ -82,8 +92,10 @@ namespace litecore {
                     if ( isVersionVector(rec) ) continue;
                     RevTreeRecord revTree(defaultKeyStore(), rec);
                     if ( newVersioning == kC4VectorVersioning ) {
+#if ENABLE_VV_UPGRADE
```

Comment:
Do we need this since if !ENABLE_VV_UPGRADE, we already downgrade newVersioning from VectorVersioning?

Reply:
It's necessary because otherwise the call to `upgradeToVersionVectors` would be a compile error, since the function is ifdef'd out.